wasmcc/wasmcc_test.go

```diff
@@ -145,6 +153,21 @@ var _ = Describe("Tests for wasmcc simple asset transfer", func() {
 				Expect(newExpectedBalAcc2).Should(Equal(newBalAcc2))
 			})
 		})
+
+		Context("error if try to transfer to wrong account", func() {
```

Comment:
Which argument makes this a transfer to a wrong account? Is it because account 3 doesn't exist? If that is the case, indicate that in the context

Reply:
Line 163: "account3". I will add a comment

Reply:
fixed